makelib/helm.mk

```diff
@@ -46,6 +46,11 @@ HELM := $(TOOLS_HOST_DIR)/helm-$(HELM_VERSION)
 # remove the leading `v` for helm chart versions
 HELM_CHART_VERSION := $(VERSION:v%=%)
 
+#Chart Museum 
```

Comment:
do you want to leave these commented out variables? if so, i'd encourage some more guidance/info about what they are and how they are expected to be used.

note also that this is a public repository, so anything added here should hopefully be generally applicable and relevant.

Reply:
added more info on how it can be used